src/marqo/tensor_search/on_start_script.py

```diff
@@ -259,7 +258,34 @@ def run(self):
         for message in messages:
             self.logger.info(message)
         self.logger.info("completed prewarming patch models")
-            
+
+
+class CheckNLTKTokenizers:
+    """Check if NLTK tokenizers are available, if not, download them.
+
+    NLTK tokenizers are included in the base-image, we do a sanity check to ensure they are available.
+    """
+    def run(self):
+        try:
+            nltk.data.find("tokenizers/punkt")
```

Comment:
this is for old version only, right? new version uses `tokenizers/punkt`.

Reply:
The newer version requires `tokenizers/punkt_tab`, which is added below.

Reply:
Sorry, I mean why do we still need to support the old version?

Reply:
We need both. They are both needed here. Previously `punkt_tab` was part of `punkt` and then it was move out of `punkt` and we have to download both of them.